src/main/resources/application.properties

```diff
@@ -61,6 +61,8 @@ finrem.notification.assignToJudge=/assign-to-judge
 finrem.notification.consentOrderMade=/consent-order-made
 finrem.notification.consentOrderNotApproved=/consent-order-not-approved
 finrem.notification.consentOrderAvailable=/consent-order-available
+finrem.notification.consentOrderAvailableCtsc=/consent-order-available-ctsc
+finrem.notification.ctscEmail=${CTSC_EMAIL:fr_applicant_sol@sharklasers.com}
```

Comment:
do we need a story or task for release for setting this to the courts address?

Reply:
nevermind, i see it in cnp flux config for prod 👍

Reply:
(see Flux PR below)